Automattic/wp-calypso
# Update metrics presented on import flow

## Proposed Changes

* Overhaul of the metrics presented on the import flow, presenting more real and relevant numbers and copy.

Discussion and more context: p1715972644965929-slack-C02NWDZBL0H

#### Default items

<img width="347" alt="Screenshot 2024-05-24 at 12 31 15" src="https://github.com/Automattic/wp-calypso/assets/876340/38e0166e-37ca-4855-9df2-aacd73fd3009">

#### Dynamic items

<img width="348" alt="Screenshot 2024-05-24 at 12 32 10" src="https://github.com/Automattic/wp-calypso/assets/876340/165874c4-511d-421b-9a83-e6855a90667e">

## Why are these changes being made?


* Some of the previous numbers and copy weren't very accurate.

## Testing Instructions



* Checkout branch or use the live link.
* Navigate to: `/setup/site-migration/site-migration-upgrade-plan?siteSlug={FREE_SITE_SLUG}&from=https://unknown.unknown&option=everything`.
* See the defaults, and check if it makes sense.
* Manually update the line `siteMetricData?.basic?.fid?.value` in the codebase to `150` (a not good FID).
* Navigate to: `/setup/site-migration/site-migration-upgrade-plan?siteSlug={FREE_SITE_SLUG}&from=https://www.mazdausa.com&option=everything`.
* Check that the dynamic information was displayed and it makes sense.

## Pre-merge Checklist
